sdk/python/feast/embedded_go/online_features_service.py

```diff
@@ -0,0 +1,128 @@
+import warnings
+from typing import Any, Dict, List, Optional, Union
+
+import pyarrow as pa
+from pyarrow.cffi import ffi
+
+from feast.errors import FeatureNameCollisionError
+from feast.feature_service import FeatureService
+from feast.online_response import OnlineResponse
+from feast.protos.feast.serving.ServingService_pb2 import GetOnlineFeaturesResponse
+from feast.protos.feast.types.Value_pb2 import RepeatedValue, Value
+from feast.repo_config import RepoConfig
+from tests.unit.test_proto_json import FeatureVector
+
+try:
+    from .lib.embedded import (
+        DataTable,
+        NewOnlineFeatureService,
+        OnlineFeatureServiceConfig,
+    )
+    from .lib.go import Slice_string
+except ImportError:
+    warnings.warn("Couldn't import embedded go library")
```

Comment:
Should this be an exception? What happens if the import fails? I see `Datatable` being used below which will cause an exception there.